collect_app/src/androidTest/java/org/odk/collect/android/support/matchers/RecyclerViewMatcher.java

```diff
@@ -69,4 +65,17 @@ public boolean matchesSafely(View view) {
             }
         };
     }
+
+    public static Matcher<View> withListSize(int listId, final int size) {
+        return new TypeSafeMatcher<View>() {
+            @Override public boolean matchesSafely(final View view) {
+                RecyclerView recyclerView = view.getRootView().findViewById(listId);
```

Comment:
Will the `View` pass here not be your `RecyclerView` (assuming you give `onView` the right matchers)?

Reply:
Right, fixed.